### DISCUSSION TITLE
Please Help

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#6432] [Q&A] [answered]
Suddenly i got this error : Call to a member function handle() on null
Please need help ASAP

**Comment by duncanmcclean** [comment, marked as answer]
Run `php artisan cache:clear` - does that help?

**Reply by HazemMohamed252** [reply]
> Run `php artisan cache:clear` - does that help?

It has been solved after i logout and login again. Thanks bro
